src/views/SystemIntake/Page1/index.tsx

```diff
@@ -0,0 +1,309 @@
+                            if (e.target.checked) {
+                              arrayHelpers.push({
+                                name: e.target.value,
+                                collaborator: ''
+                              });
+
+                              // Check parent radio if it's not already checked
+                              if (
+                                !formikProps.values.governanceTeams.isPresent
+                              ) {
+                                formikProps.setFieldValue(
+                                  'governanceTeams.isPresent',
+                                  true
+                                );
+                              }
+                            } else {
+                              const index = formikProps.values.governanceTeams.teams
+                                .map(t => t.name)
+                                .indexOf(e.target.value);
+
+                              arrayHelpers.remove(index);
+                            }
+                          }}
+                          value={team.value}
+                        />
+                        {formikProps.values.governanceTeams.teams.map(
+                          (t, index) => {
+                            if (team.value === t.name) {
+                              const id = t.name.split(' ').join('-');
+                              return (
+                                <div
+                                  key={`${id}-Collaborator`}
+                                  className="width-card-lg margin-top-neg-2 margin-left-3 margin-bottom-2"
+                                >
+                                  <Field
+                                    as={TextField}
+                                    id={`IntakeForm-${id}-Collaborator`}
+                                    label="Collaborator Name"
+                                    maxLength={50}
+                                    name={`governanceTeams.teams[${index}].collaborator`}
+                                  />
+                                </div>
+                              );
+                            }
+                            return null;
+                          }
+                        )}
+                      </Fragment>
+                    );
+                  })}
+                </>
+              )}
+            </FieldArray>
+          </div>
+          <Field
+            as={RadioField}
+            checked={formikProps.values.governanceTeams.isPresent === false}
+            id="IntakeForm-YesGovernanceTeam"
+            name="governanceTeams.isPresent"
+            label="None of the above"
+            onChange={() => {
+              formikProps.setFieldValue('governanceTeams.isPresent', false);
+              formikProps.setFieldValue('governanceTeams.teams', []);
+            }}
+            value={false}
+          />
+        </fieldset>
```

Comment:
I wonder if there's not a way to break this out for readability... can we make each form component its own class? Even its own const in this file? Would make things a lot less overwhelming. Let's talk this one over.